packages/api/src/routes/internal/medical/organization.ts

```diff
@@ -30,6 +30,7 @@ router.put(
   asyncHandler(async (req: Request, res: Response) => {
     if (Config.isSandbox()) return res.sendStatus(httpStatus.NOT_IMPLEMENTED);
     const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const overrideProviderCheck = getFromQueryAsBoolean("overrideProviderCheck", req);
```

Comment:
Can you add a TSDoc explaining when we might want to use this parameter, please?